### ISSUE TITLE
Collapsible cities buttons open ALL cities in Mapping Reports

### ISSUE THREAD

**Issue by Hare-J** [abaporu-C/GROW-POS#66]
On the Azure site when you go to Mapping Reports, if you click any of the cities like Welland, Niagara Falls, etc. it will open ALL collapsible tables instead of just the one clicked.

**Comment by Lmarquez1983** [comment]
Yes, this is basically because the code that shows the city dynamically is encapsuled with the collapsable. I haven't found the way to get it by city
